docs/wiki/development/ace3-config-entries.md

```diff
@@ -49,6 +49,7 @@ ace_recoil_shakemultiplier
 ace_overpressure_angle
 ace_overpressure_range
 ace_overpressure_damage
+ace_overheating_closedbolt
```

Comment:
What happened with uppercasing in this section.

Reply:
no idea, I only added this one entry. I can camel case all of them if you want though

Reply:
Eeh, separate PR if anything I'd say.